types/index.d.ts

```diff
@@ -37,26 +37,37 @@ declare global {
         /**
          * Array representing the current selection plus any locked items that the user has attempted to select.
          */
-        itemsIncludingLocked: Array<SceneNode>;
+        readonly itemsIncludingLocked: Array<SceneNode>;
         /**
          * True if the selection isn’t empty and consists of one or more non-Artboards. Never true at the same time as hasArtboards.
          */
-        hasArtwork: boolean;
+        readonly hasArtwork: boolean;
         /**
          * True if the selection isn’t empty and consists of one or more Artboards. Never true at the same time as hasArtwork.
          */
-        hasArtboards: boolean;
+        readonly hasArtboards: boolean;
         /**
          * The context in which selection and edit operations must occur. If the user hasn’t drilled into any container node, this value is the document root, and its scope includes all immediate children of the pasteboard (including Artboards), and all immediate children of all those Artboards.
          */
-        editContext: SceneNode;
+        readonly editContext: SceneNode;
         /**
-         * The preferred parent to insert newly added content into. Takes into account the current edit context as well as the “focused artboard” if in the root context.
+         * The preferred parent to insert newly added content into. Takes into account the current edit context as well as the "focused artboard" if in the root context.
+         Typically this is the same parent where, for example, XD's shape drawing tools would add items.
+         *
+         * _Selected items are not necessarily all immediate children of the `insertionParent`._ They can be anywhere within the [edit context's](/reference/core/edit-context.md) scope.
```

Comment:
There's a link in here that should be removed.

Consider doing a pass on link removal as I've seen several others throughout the docs as well.

This seems to be a simple copy-paste issue.

```suggestion
         * _Selected items are not necessarily all immediate children of the `insertionParent`._ They can be anywhere within the edit context's scope.
```